CondCore/HLTPlugins/test/test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Save current working dir so img can be outputted there later
+W_DIR=$(pwd);
+# Set SCRAM architecture var
+SCRAM_ARCH=slc6_amd64_gcc630;
+export SCRAM_ARCH;
+source /afs/cern.ch/cms/cmsset_default.sh;
+eval `scram run -sh`;
+# Go back to original working directory
+cd $W_DIR;
+# Run get payload data script
+
+mkdir -p $W_DIR/plots
+
+####################
+# Test display 
+####################
+/afs/cern.ch/user/c/condbpro/public/BROWSER_PI/getPayloadData.py \
```

Comment:
hi @mmusich - how can this afs dependency be avoided? (i gather getPayloadData.py is not in cmssw?)

Reply:
hello @davidlange6.
Just to clarify: this is not a real dependence, as it the script is not run in integration tests, since the bot user would need to be granted reading credentials for oracle; it's more a convenience script for developers.
In any case I would be more than happy to remove this afs dependency, e.g. by committing the script to cmssw (it was already discussed here: https://github.com/cms-sw/cmssw/pull/19874#discussion_r130380469). I think the ball is in @ggovi's court about this.

Reply:
@ggovi confirms he is going to add this script...

Reply:
Thanks a lot. As soon as it can be picked up from release in 10.0.x IBs I'll clean this up here and everywhere else.

Reply:
sorry for the misunderstanding, we could obviously add the script in the CMSSW repository, however its usage will be limited to the db admins, those owning the Oracle passwords.